### DISCUSSION TITLE
platformio-ide-terminal not working in atom

### DISCUSSION THREAD

**Discussion by thuycnguyen** [atom/atom#22241] [Q&A] [answered]
I've uninstalled and reinstalled platformio-ide-terminal and the window still won't show up. Please help.

**Comment by suhaibrabei** [comment]
``

**Comment by suhaibrabei** [comment]
[](url)

**Comment by suhaibrabei** [comment]
# @22172

**Comment by UziTech** [comment, marked as answer]
Platformio-ide-terminal is [no longer maintained](https://github.com/platformio/platformio-atom-ide-terminal/issues/543). Please uninstall platformio-ide-terminal and install either [terminus](https://atom.io/packages/terminus) or [x-terminal](https://atom.io/packages/x-terminal)
